test/reshape.jl

```diff
@@ -499,12 +498,36 @@ end
     @test unstack(df, [:id, :id2], :var, :val) == unstack(df, :var, :val) ==
           DataFrame(id=1:3, id2=1:3, a=1:3:7, b=2:3:8, c=3:3:9)
 
+    # an exertice on current unstack invariants
     Random.seed!(1234)
-    for i in 1:3
+    for i in 1:16
```

Comment:
Why so high?

Reply:
to ensure that `[1,1,1,2,2,2,3,3,3]` is reordered in such a way that we test several first orders of appearance other than 1,2,3, as e.g. reordering to `[1,2,3,1,2,3,1,2,3]` does not hit the "hard case" where order of first appearance is broken. I have checked that this is very fast (sub second) anyway (the whole testset for reshape takes 1 minute so this is negligible).